app/assets/stylesheets/fontawesome_icons.sass

```diff
@@ -24,6 +24,23 @@
   vertical-align: -10%
   font-size: 1.3333333333333333em
 
+#commits_index_page,
```

Comment:
Could we please move the custom styles to a different file (new or existing one) such that in the future when we upgrade the font-awesome lib, the changes wouldn't be lost.